app/Controllers/configureController.php

```diff
@@ -241,13 +241,16 @@ public function archivingAction() {
 	 * checking if categories and feeds are still in use.
 	 */
 	public function queriesAction() {
```

Comment:
That's a good idea. I think that we should move gradually, while introducing unit tests. This way refactoring would be easier.